packages/hardhat-core/scripts/test-run-forked-block.ts

```diff
@@ -30,7 +30,7 @@ async function main(rpcUrl: string, blockNumber: bigint) {
   const remoteCommon = new Common({ chain: chainId });
   const hardfork = remoteCommon.getHardforkByBlockNumber(blockNumber);
```

Comment:
I think we can remove this line (yes, I know, the linter should run in this file, but that turns out to be surprisingly hard :disappointed:)

Reply:
addressed in https://github.com/NomicFoundation/hardhat/pull/3953/commits/724b96c4e94c1e2d227bac7bd42d5b46972fe8a8